icepyx/core/query.py

```diff
@@ -19,6 +18,7 @@
 # from icepyx.core.granules import Granules
 from icepyx.core.variables import Variables as Variables
 import icepyx.core.validate_inputs as val
+from icepyx.core.auth import EarthdataAuthMixin
```

Comment:
I know the list is currently not up to snuff on this, but can we put this in what would-be ABC order (so after APIformatting)?

Reply:
sure, happy to follow alphabetical as a standard!